packages/gestalt/src/Fieldset.js

```diff
@@ -0,0 +1,34 @@
+// @flow strict
+import type { Node } from 'react';
+import cx from 'classnames';
+import Text from './Text.js';
+import labelStyles from './Label.css';
+import formStyles from './FormElement.css';
+import formLabelStyles from './FormLabel.css';
+import boxWhitespaceStyles from './boxWhitespace.css';
+import whitespaceStyles from './Whitespace.css';
+import boxStyles from './Box.css';
+
+type Props = {|
+  children: Node,
+  legend: string,
+  legendDisplay?: 'visible' | 'hidden',
+|};
+
+export default function Fieldset({ legend, legendDisplay = 'visible', children }: Props): Node {
+  return (
+    <fieldset className={cx(formStyles.unstyled, whitespaceStyles.p0, whitespaceStyles.m0)}>
```

Comment:
whitespaceStyles.p0 could we use here `boxWhitespaceStyles.paddingX0` instead of `whitespaceStyles.p0`, just wondering, haven't played with the code, so it might change the style, but it should be the same right?